### PR TITLE
Tests/master/modules 3138

### PR DESCRIPTION
**Pull request by ThoughtCrhyme**
Removed r-spec tests and fixed beaker acceptance test.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -19,4 +19,36 @@ desc 'Generate code coverage'
...
\ No newline at end of file
+end
+
+desc 'Executes reference tests (agent only) intended for use in CI'
+task :reference_tests do
+    command =<<-EOS
+bundle exec beaker                          \
+    --debug                                 \
+    --preserve-hosts never                  \
+    --config tests/configs/$PLATFORM        \
```
Comment (cowofevil) [inline Rakefile]: What happens if $PLATFORM is not set?

File: Rakefile [hunk 2]
```diff
@@ -19,4 +19,36 @@ desc 'Generate code coverage'
...
+    sh command
+end
+
+desc 'Executes accetpance tests (master and agent) indened for use in CI'
+task :acceptance_tests do
+    command =<<-EOS
+bundle exec beaker
+    --debug                                 \
+    --preserve-hosts never                  \
+    --config tests/configs/$PLATFORM        \
```
Comment (cowofevil) [inline Rakefile]: What happens if $PLATFORM is not set?

File: Rakefile (hunk 1 above)
Comment (cowofevil) [inline Rakefile]: Let's make sure the task will fail with a descriptive message if the $PLATFORM envar is not set.

File: Rakefile (hunk 2 above)
Comment (cowofevil) [inline Rakefile]: Let's make sure the task will fail with a descriptive message if the $PLATFORM envar is not set.

**Comment by cowofevil** [comment]
@ThoughtCrhyme See [PR](https://github.com/ThoughtCrhyme/puppetlabs-chocolatey/pull/1) to fix Travis CI failure.

File: Rakefile (hunk 2 above)
Comment (ThoughtCrhyme) [inline Rakefile]: Done, see lines 56-62 below.

File: Rakefile [hunk 3]
```diff
@@ -19,4 +19,44 @@ desc 'Generate code coverage'
...
+    --pre-suite tests/acceptance/pre-suite  \
+    --tests tests/acceptance/tests
+    EOS
+    sh command
+end
+
+task :acceptance_tests => [:blah]
+task :reference_tests => [:blah]
+
+task :blah do
```
Comment (cowofevil) [inline Rakefile]: This should have a more descriptive name and maybe a comment on how it is triggered.

File: Rakefile [hunk 4]
```diff
@@ -19,4 +19,44 @@ desc 'Generate code coverage'
...
+desc 'Executes accetpance tests (master and agent) indened for use in CI'
+task :acceptance_tests do
+    command =<<-EOS
+bundle exec beaker
+    --debug                                 \
+    --preserve-hosts never                  \
+    --config tests/configs/$PLATFORM        \
+    --keyfile ~/.ssh/id_rsa-acceptance      \
+    --load-path tests/lib                   \
+    --type aio                              \
```
Comment (cowofevil) [inline Rakefile]: This option should be removed for acceptance tests.

File: Rakefile [hunk 5]
```diff
@@ -19,4 +19,44 @@ desc 'Generate code coverage'
...
+    --pre-suite tests/reference/pre-suite   \
+    --tests tests/reference/tests
+    EOS
+    sh command
+end
+
+desc 'Executes accetpance tests (master and agent) indened for use in CI'
+task :acceptance_tests do
+    command =<<-EOS
+bundle exec beaker
```
Comment (cowofevil) [inline Rakefile]: Missing "\" at end of line.

File: Rakefile [hunk 6]
```diff
@@ -19,4 +19,44 @@ desc 'Generate code coverage'
...
+    --load-path tests/lib                   \
+    --type aio                              \
+    --pre-suite tests/reference/pre-suite   \
+    --tests tests/reference/tests
+    EOS
+    sh command
+end
+
+desc 'Executes accetpance tests (master and agent) indened for use in CI'
+task :acceptance_tests do
```
Comment (cowofevil) [inline Rakefile]: The acceptance tests also require that "BEAKER_PE_DIR" or "pe_dist_dir" envar be set.

File: Rakefile [hunk 7]
```diff
@@ -19,4 +19,44 @@ desc 'Generate code coverage'
...
+    --tests tests/acceptance/tests
+    EOS
+    sh command
+end
+
+task :acceptance_tests => [:blah]
+task :reference_tests => [:blah]
+
+task :blah do
+    abort("PLATFORM variable not present, aborting test.") unless ENV["PLATFORM"]
```
Comment (cowofevil) [inline Rakefile]: Acceptance and reference tests also require that "MODULE_VERSION" envar be set.

File: Rakefile (hunk 3 above)
Comment (ThoughtCrhyme) [inline Rakefile]: Hah! Meant to change that before push

**Pull request merged**